src/Helper/DateTimeFormatter.php

```diff
@@ -106,7 +106,10 @@ public function getDatetime($data, ?string $timezone = null): \DateTime
         }
 
         if ($data instanceof \DateTimeImmutable) {
-            return \DateTime::createFromFormat(\DateTime::ATOM, $data->format(\DateTime::ATOM));
+            $dateTime = \DateTime::createFromFormat(\DateTime::ATOM, $data->format(\DateTime::ATOM));
```

Comment:
why not DateTime::createFromImmutable or if bumping to 8.0 anyway,
why not DateTime::createFromInterface?

Reply:
yeah i didn't changed the code either so much when doing the DateTimeFormatter ;P